atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -117,18 +116,20 @@ private void createOverflowTable() {
     }
 
     private void putTableNameMapping(String fullTableName, String shortTableName) {
-        try {
-            conns.get().insertOneUnregisteredQuery(
-                    "INSERT INTO " + AtlasDbConstants.ORACLE_NAME_MAPPING_TABLE
-                            + " (table_name, short_table_name) VALUES (?, ?)",
-                    fullTableName,
-                    shortTableName);
-        } catch (PalantirSqlException ex) {
-            if (ex.getMessage().contains(ORACLE_UNIQUE_CONSTRAINT_ERROR)) {
-                dropTableInternal(fullTableName, shortTableName);
+        if (config.useTableMapping()) {
```

Comment:
This is fine, but I find it very odd that the mapper object doesn't own the table.  It's odd to see logic for inserting into the ORACLE_NAME_MAPPING_TABLE outside of the mapping classes.